src/test/java/com/atlassian/db/replica/api/TestConsistency.java

```diff
@@ -137,22 +136,6 @@ public void shouldNotRefreshAfterSelectForUpdate() throws SQLException {
         verify(consistency, never()).write(any());
     }
 
-    @Test
-    public void shouldSetAutoCommitRefreshWhenInTransaction() throws SQLException {
```

Comment:
If  test is deleted, shouldn't a new one be created for reusing transactions 🤔

Reply:
There are two new tests in `DualConnectionIT`.